webapp-django/crashstats/supersearch/tests/test_views.py

```diff
@@ -548,6 +571,62 @@ def mocked_get(url, params, **options):
         )
         eq_(response.status_code, 200)
 
+    @mock.patch('crashstats.crashstats.models.Bugs.get')
+    @mock.patch('requests.get')
+    def test_search_results_ratelimited(self, rget, cpost):
+        def mocked_post(**options):
+            return {
+                "hits": [],
+                "total": 0
+            }
+
+        def mocked_get(url, params, **options):
+            assert 'supersearch' in url
+
+            if 'supersearch/fields' in url:
+                return Response(SUPERSEARCH_FIELDS_MOCKED_RESULTS)
+
+            if 'product' in params and 'WaterWolf' in params['product']:
+                return Response({
+                    "hits": [],
+                    "facets": {},
+                    "total": 0
+                })
+            elif 'product' in params and 'SeaMonkey' in params['product']:
+                return Response({
+                    "hits": [],
+                    "facets": {},
+                    "total": 0
+                })
+            elif (
+                'signature' in params and
+                '~nsASDOMWindowEnumerator' in params['signature']
+            ):
+                return Response({
+                    "hits": [],
+                    "facets": {},
+                    "total": 0
+                })
+            else:
+                return Response({"hits": [], "facets": [], "total": 0})
```

Comment:
You can keep this line only, there's no need to have different blocks for products and stuff.